config/notifiers.go

```diff
@@ -625,6 +626,24 @@ func (c *OpsGenieConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 		}
 	}
 
+	for _, v := range c.VisibleTo {
+		if v.ID == "" && v.Username == "" && v.Name == "" {
+			return fmt.Errorf("opsGenieConfig visible_to %v has to have at least one of id, username or name specified", v)
+		}
+
+		if strings.Contains(v.Type, "{{") {
```

Comment:
I'm afraid this check doesn't work. Consider the following example:

```
this is an invalid template }}
```

I would just remove this if statement and check all VisibleTo's using `template.New("").Parse()`?

Reply:
I suppose you could also check for `strings.Contains(v.Type, `}}`)?